service/storage_proxy.cc

```diff
@@ -2319,6 +2319,13 @@ future<> storage_proxy::send_hint_to_endpoint(frozen_mutation_and_schema fm_a_s,
             allow_hints::no);
 }
 
+future<> storage_proxy::mutate_hint_from_scratch(frozen_mutation_and_schema fm_a_s) {
```

Comment:
What does "from_scratch" means?

Reply:
In this context it means that the whole effort of writing this mutation to all replicas is repeated. I used the terminology from the log message above the place where this function is called.

Reply:
I think more descriptive name that make it clear that a mutation is sent to all owning replicas would be better.

Reply:
Any particular name you have in mind @gleb-cloudius ?

Reply:
`send_hint_to_all_replicas()`

Reply:
Like with the timeout, I think we can rename this function on master and then backport the change (on master it is called the same as here). Leaving the function name as it is here will make life easier when backporting.

Reply:
Backporting what? This is a backport already.

Reply:
This is a limited backport of https://github.com/scylladb/scylla/commit/01084a79b8f47f63aeb54bd110135f3850ad3fc8

If we change the name here then it will cause conflicts for future backports. As you correctly noticed @gleb-cloudius, this is already a backport so maybe we shouldn't change too much. If we want to change something unrelated then it's better to submit it to master and backport again.

Reply:
We will definitely not backport function name change. Well I guess it is my fault I did not complaine about the name on the original submission (although I do remember scratching my head when I saw the function name).

Reply:
You have a fair point so let's rename on master.

Reply:
Understood. In that case, I changed the name to `send_hint_to_all_replicas`, as you suggested.

Reply:
But please send the rename to the master as well.